styles/common.tsx

```diff
@@ -210,3 +212,40 @@ export const Tooltip = styled(Tippy)`
 	border: 0.5px solid ${(props) => props.theme.colors.navy};
 	border-radius: 4px;
 `;
+
+export const SolidTooltip = styled(Tooltip).attrs({
+	trigger: 'click',
+	arrow: false,
+	interactive: true,
+})`
+	width: 150px;
+	.tippy-content {
+		padding: 0;
+	}
+`;
+
+export const SolidTooltipContent = styled.div`
+	padding: 16px 0 8px 0;
+`;
+
+export const SolidTooltipCustomValueContainer = styled.div`
+	margin: 0 10px 5px 10px;
+`;
+
+export const SolidTooltipCustomValue = styled(NumericInput)`
+	width: 100%;
+	border: 0;
+	font-size: 12px;
+	::placeholder {
+		font-family: ${(props) => props.theme.fonts.mono};
+	}
+`;
+
+export const SolidTooltipItemButton = styled(Button)`
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding-left: 10px;
+	padding-right: 10px;
+`;
```

Comment:
It's possible to create a "SolidTooltip" component in the future that has these... but for now, I went for something quick and reusable (we could use it in the GasPriceTooltip etc)